homeassistant/components/zoneminder.py

```diff
@@ -40,6 +41,29 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+class _Feature:
+    def __init__(self, version_str):
+        self._version = packaging.version.parse(version_str)
+
+    def is_supported(self):
+        """Return the remote instance support for this feature.
+
+        Returns False if this feature isn't supported on the currently
+        connected zoneminder instance. Otherwise returns True.
+        """
+        remote_version = ZM.get('version', None)
```

Comment:
Yes, `hass.data` is the right approach. 

I also like `zoneminder.alarm_status_supported()` idea !